queries/fortran/highlights.scm

```diff
@@ -0,0 +1,92 @@
+;; by @oponkork
+
+(identifier) @variable
+(string_literal) @string
+(number_literal) @number
+(boolean_literal) @boolean
+(comment) @comment
+
+[
+  (intrinsic_type)
+  ; (primitive_type_qualifier)
+  "dimension"
+  "intent"
+  "in"
+  "out"
+  "inout"
+  "type"
+  "endtype"
+  "attributes"
+  "global"
+  "device"
+  "host"
+  "grid_global"
+] @type
+
+[
+  "bind"
+  "call"
+  "contains"
+  "continue"
+  "cycle"
+  "endfunction"
+  "endmodule"
+  "endprogram"
+  "endsubroutine"
+  "enumerator"
+  "equivalence"
+  "exit"
+  "format"
+  "function"
+  "goto"
+  "include"
+  "interface"
+  "module"
+  "parameter"
+  "print"
+  "program"
+  "read"
+  "return"
+  "stop"
+  "subroutine"
+  "use"
+  "write"
+  "implicit"
+  (none)
```

Comment:
Maybe `@constant.builtin`?

Reply:
I think `none` is a void return type in Fortran